docs/static/troubleshoot/ts-kafka.asciidoc

```diff
@@ -163,4 +163,19 @@ https://github.com/logstash-plugins/logstash-input-kafka/issues/210
 * If (and only if) upgrading is not possible, set `consumer_threads` to `1`.
 
 
+[discrete] 
+[[ts-kafka-kerberos-debug]]
+===== Debug Kerberos SASL
+
+To troubleshoot authentication failures in Kafka client, add `-Dsun.security.krb5.debug=true` to `jvm.options` and the following lines to `log4j2.properties`:
+
+```
+logger.kafkainput.name = logstash.inputs.kafka
+logger.kafkainput.level = debug
+logger.kafkaoutput.name = logstash.outputs.kafka
+logger.kafkaoutput.level = debug
+logger.kafka.name = org.apache.kafka
+logger.kafka.level = debug
+```
 
+NOTE: Logging entries for Kerberos are NOT sent through Log4j but go directly to the console.
```

Comment:
```suggestion
[discrete] 
[[ts-kafka-kerberos-debug]]
===== Debug Kerberos SASL

You can set up your machine to help you troubleshoot authentication failures in the Kafka client.

* In `config/jvm.options`, add:
+
[source,txt]
-----
`-Dsun.security.krb5.debug=true` to `jvm.options` 
-----

* In `config/log4j2.properties`, add: 
+
[source,txt]
-----
logger.kafkainput.name = logstash.inputs.kafka
logger.kafkainput.level = debug
logger.kafkaoutput.name = logstash.outputs.kafka
logger.kafkaoutput.level = debug
logger.kafka.name = org.apache.kafka
logger.kafka.level = debug
-----

NOTE: Logging entries for Kerberos are NOT sent through Log4j but go directly to the console.
```

Reply:
Proposing a formatting tweak to make sure that the inline `jvm.options` setting doesn't get overlooked, and to make both settings more eye-catching.

Reply:
Can we give users more info on next steps or what to look for? Right now, we're helping them set up debugging and then dropping them.

Reply:
We mainly want to write down how to enable debug log to check kerberos issues as it is tricky that the log does not send to log4j log. The next step is that user needs to get the hints from debug log. Do you think we need to add a line to remind them to check the log?